include/language.php

```diff
@@ -628,16 +630,17 @@ public function get_locale( $filter = 'raw' ) {
 	 *
 	 * @since 3.4
 	 *
-	 * @param bool $raw Whether or not properties should be raw. Default to `false`.
+	 * @param string $context Whether or not properties should be filtered. Accepts `raw` or `display`.
+	 *                        Default to `display` which filters some properties.
```

Comment:
When I read this, I'm not sure that `'display'` is the best. Maybe `'filtered'`?

Reply:
Idea: `'not-raw'`! (kidding)

Reply:
I suggest `'db'` instead of ``raw'` as this is really in this context that we want this method to return the unfiltered url.